CMakeLists.txt

```diff
@@ -184,6 +185,22 @@ if(MI_USE_CXX)
   endif()
 endif()
 
+if(NOT MI_DISABLE_REDIRECT)
+  if(MSVC AND MSVC_C_ARCHITECTURE_ID MATCHES "ARM")
+    set(MI_DISABLE_REDIRECT ON)
+    message(WARNING "Disabling redirection lib is required on ARM (MI_DISABLE_REDIRECT=ON)")
```

Comment:
Just personal opinion but would it be too much to add a comment in the git message or here why is that ?

Reply:
Not too much to ask at all. I'll add a comment when I can

Reply:
Hopefully that verbiage is sufficient